extra-python/distlib/autobuild/defines

```diff
@@ -0,0 +1,7 @@
+PKGNAME=distlib
+PKGSEC=python
+PKGDES="a library which implements low-level functions that relate to packaging and distribution of Python software"
```

Comment:
Capitalise first letter. Phrasing could be better, use this:

"A Python library that implements low-level functions for packaging and redistribution"